### ISSUE TITLE
Use Gleam.io to distribute GRC & increase attention towards GRC social media

### ISSUE THREAD

**Issue by grctest** [Erkan-Yilmaz/Gridcoin-tasks#75]
https://gleam.io/

Perhaps we can get away with the free version, unless only 100 users can participate? https://gleam.io/pricing#

Their pricing for PRO is on an individual site basis!

We could use this to drive up Twitter, SoundCloud, YouTube subscriber/follower quantities. This would be pretty good if steemit was supported.

---

Social media accounts to promote:
https://www.youtube.com/channel/UC0Z3uczcC2okYRrLLCZFrMQ (We can only get a custom URL once we reach 100 subscribers!)
https://soundcloud.com/gridcoin-community-hangouts
https://gridcoin.tumblr.com/
https://www.fb.com/gridcoins
http://www.reddit.com/r/gridcoin
https://twitter.com/GridcoinNetwork

**Comment by Erkan-Yilmaz** [comment]
mentioned in [hangout 7](https://github.com/Erkan-Yilmaz/Gridcoin-hangout-minutes/blob/master/hangout_2016_09_17.MD): for doing contests, ...

**Comment by LokiToki** [comment]
I checked gleam.io and with free account you're not allowed to add a picture. Free account is very limited and close to useless for semi-pro use.

**Comment by grctest** [comment]
$39 for a test month isn't too bad, could be worth it if we get several hundred entries (subscriptions across multiple websites).

**Comment by LokiToki** [comment]
I could start some free stuff to familiarize myself with the system or is someone already into it? Do i have the "permission" to start gridcoin ads with gleam.io? ;) We could go for a paid month after some test...

**Comment by gustav1976** [comment]
GRCnation is already working with gleam.io and they are the official Gridcoin marketing partner for Europe. But it would be cool to see additional concepts. Gridcoin could only benefit from more ideas.

**Comment by grctest** [comment]
> @gustav1976 
> GRCnation is already working with gleam.io and they are the official Gridcoin marketing partner for Europe. But it would be cool to see additional concepts. Gridcoin could only benefit from more ideas.

How/where have you been utilizing gleam? There's nothing on [grcnation's Twitter](https://twitter.com/GRCgermany) about it, nor is there anything on grcnation.com other than a couple old client install guides..

Is there a blog or a thread following what official marketing you've undertaken on behalf of Gridcoin (not for uscore/grcnation)? Is there actually an official partner program in Gridcoin? It seems more of an unofficial position.

**Comment by LokiToki** [comment]
I wrote a mail to gridcoins@gmail.com (official GRCnation mail address) 6 days ago (03-16) and got no answer. For me it's obvious that they are not longer active and i keep on investigating gleam.io system.

If anybody is interested i could set up an User-Account in my gleam Gridcoin-Site (like a project). I plan to to a test competition and spend some GRC or so as a reward for participants.

**Comment by LokiToki** [comment]
I did a quick Competition. The major problem was to find easy entry points for absolutely inexperienced people. Something like "What is this shi*about? And why i'm here?". I found two acceptable youtube videos. To be honest: just going to soundcloud and listening to last hangout is confusing to new member. Reading latest post on twitter or facebook also. 

So the first step before we start promoting GRC is to set up something like a twitter account "GridcoinStarter" or make a easy explaining one-pager and so on. 

Sry, but it's hard to make a "Oh, look at this cool stuff!" without digestible material. Here now the test competition:
https://gleam.io/hI3Aj/investigate-gridcoin-and-science-win-real-money

Greetings
